src/polygon/src/polygon.cpp

```diff
@@ -727,12 +728,17 @@ class EarClip {
       }
       if (Clipped(v)) {
         start = v->right->left;
+        if (!Clipped(start)) {
+          ProcessEar(start);
+          ++numTri;
```

Comment:
Here was the real source of the seg fault - I had fixed this loop in FindStarts, but failed to make a similar fix here. I also found a better way, so I updated them both. I was failing to process the start vert if it had been updated, and I was also getting an incorrect triangle count. I believe the second one was the error causing the seg fault, as numTri is what's used to terminate the ear clipping loop.